docs/src/main/asciidoc/rest-json.adoc

```diff
@@ -245,6 +245,151 @@ public class CustomObjectMapper {
 }
 ----
 
+There is one unqualified `ObjectMapper` instance provided by the quarkus-jackson extension.  Your code may produce two additional
+`ObjectMapper` instances. The client `ObjectMapper` instance is qualified by the `@RestClientObjectMapper` annotation. The server
+`ObjectMapper` instance is qualified by the `@RestServerObjectMapper` annotation.
+
+If the client `ObjectMapper` instance is available, it is used by the quarkus-jaxrs-client-reactive extension for serialization
+and deserialization.  If the client instance is not provided, then quarkus-resteasy-reactive uses the unqualified `ObjectMapper`
+instance.
+
+If the server `ObjectMapper` is available, it is used by the quarkus-resteasy-reactive extension for serialization and
+deserialization.  If the server instance is not provided, then quarkus-resteasy-reactive uses the unqualified `ObjectMapper`
+instance.
+
+You can produce, inject, and customize each of these instances independently.  The following snippet demonstrates how to inject
+each instance using the `@RestClientObjectMapper` and `@RestServerObjectMapper` annotations.
+
+[source,java]
+----
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.quarkus.resteasy.reactive.jackson.common.RestClientObjectMapper;
+import io.quarkus.resteasy.reactive.jackson.common.RestServerObjectMapper;
+
+import javax.inject.Inject;
+
+public class UseFallback {
+
+    private final ObjectMapper serializer;
+
+    @Inject
+    public UseFallback(ObjectMapper mapper,
+            @RestClientObjectMapper Instance<ObjectMapper> clientInstance) {
+        serializer = clientInstance.isUnsatisfied() ? mapper : clientInstance.get();
+    }
+
+}
+----
```

Comment:
I'm not sure we really want to document this. These mappers are supposed to be used for a very specific purposes by the RESTEasy Reactive extension so I'm not sure I would document anything except the customization of them.

In any case, if we want to keep that (I recommend we don't), I would put it at the end, not at the start.

Reply:
Makes sense to remove the fallback in this example.  When would someone wish to inject the alternate ObjectMappers into non-quarkus code?  Possibly/Probably an edge case.